apps/src/templates/GDPRDialog.jsx

```diff
@@ -0,0 +1,82 @@
+import React, {Component, PropTypes} from 'react';
+import Button from './Button';
+import i18n from "@cdo/locale";
+import BaseDialog from './BaseDialog';
+import DialogFooter from "./teacherDashboard/DialogFooter";
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  },
+  instructions: {
+    marginTop: 20
+  }
+};
+
+export default class GDPRDialog extends Component {
+  static propTypes = {
+    isDialogOpen: PropTypes.bool.isRequired,
+    currentUserId: PropTypes.number.isRequired,
+    requestIp: PropTypes.string.isRequired,
```

Comment:
Does the client need to know anything about the IP address?  I'd have thought the receiving controller would set that.